cmake/Template_Helpers.cmake

```diff
@@ -0,0 +1,155 @@
+include_guard(GLOBAL)
+
+include(FindPackageHandleStandardArgs)
+find_package(PkgConfig REQUIRED)
+
+macro(Template_FindPackage name)
+	#[===[.md
+	# Template_FindPackage
+
+	A compatibility macro that links `find_package(CONFIG)` packages with `pkg-config`. This should only
+	be called within the `Find<PackageName>.cmake` file.
+
+	Note: Version range syntax is not supported for pkg-config searching. Only the lower bound will be respected.
+
+	]===]
```

Comment:
These I want to make available in https://github.com/LecrisUT/CmakeExtraUtils, but basically it is a compatibility function to integrate `find_package(CONFIG)` and `pkg-config`, having a fallthrough if the former fails to try the latter.